egs/tedlium3/ASR/prepare.sh

```diff
@@ -146,6 +151,9 @@ if [ $stage -le 6 ] && [ $stop_stage -ge 6 ]; then
       log "Generate data for BPE training"
       cat data/lang_phone/train.text | cut -d " " -f 2- 
       > $lang_dir/transcript_words.txt
+      sed -i 's/ <unk>//g' $lang_dir/transcript_words.txt
```

Comment:
I think you need to keep `<unk>` in the transcript for NN model training.

Reply:
Yes, probably `<unk>` should be treated as a special token, not processed by BPE but passed through.

Reply:
In the training supervision text, the `<unk>` is not be removed. I just remove it in `transcript_words.txt` which is prepared for training bpe.

Reply:
`<unk>` should be encoded as a single token in NN training. But you are currently encoding it into 4 tokens.